compiler/test/dotty/tools/dotc/coverage/CoverageTests.scala

```diff
@@ -61,6 +61,26 @@ class CoverageTests:
           val instructions = FileDiff.diffMessage(expectFile.toString, targetFile.toString)
           fail(s"Coverage report differs from expected data.\n$instructions")
 
+      // measurement files only exist in the "run" category
```

Comment:
This verifies the measurement file against the check file (if a measurement check file exists).

Mainly so that none of the other tests need tweaking